src/data/teams.json

```diff
@@ -762,6 +762,19 @@
           "profileImage": "https://ca.slack-edge.com/T07PJD1FZ-U01HR3S3WCS-810bc78e23bd-512",
           "slackUsername": "Chatrasal Singh"
         }
+      },
+       {
+        "title": "Marketing Analyst",
+        "isLead": false,
+        "payPerDay": 2800,
+        "createdDate": "2020-07-08T02:14:59.307535Z",
+        "contributor": {
+          "contributorId": "U01JHJS30SC",
+          "displayName": "Tomiwa",
+          "githubUsername": "oluwatomiwatemidayo",
+          "profileImage": "https://ca.slack-edge.com/T07PJD1FZ-U01JHJS30SC-62f5af21aed7-512",
```

Comment:
use this as his profile image url instead: `https://avatars1.githubusercontent.com/u/72953345?s=400&u=8a7d5931cd7e0041fc6e68683ad25aeb4e7016ce&v=4`

Reply:
Alright.